From b48faff0d212c5eed1f21afaaad35669f8b77878 Mon Sep 17 00:00:00 2001
From: William Wu <wulf@rock-chips.com>
Date: Wed, 5 Apr 2017 16:21:22 +0800
Subject: [PATCH] arm64: rockchip_linux_defconfig: enable CONFIG_USB_UAS

Some rockchip platforms (e.g. rk3399, rk3328) xHCI controller
support stream for UASP (USB Attached SCSI PROTOCOL), use of
UAS generally provides faster transfers compared to the older
USB Mass Storage Bulk-Only Transport (BOT) drivers.

Change-Id: I67ce5cdb821413d3c4d018be31c892d20d831470
Signed-off-by: William Wu <wulf@rock-chips.com>
---
 arch/arm64/configs/rockchip_linux_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/rockchip_linux_defconfig b/arch/arm64/configs/rockchip_linux_defconfig
index 70e34734ca89..5188542a06d3 100644
--- a/arch/arm64/configs/rockchip_linux_defconfig
+++ b/arch/arm64/configs/rockchip_linux_defconfig
@@ -416,6 +416,7 @@ CONFIG_USB_OHCI_HCD=y
 CONFIG_USB_OHCI_HCD_PLATFORM=y
 CONFIG_USB_ACM=y
 CONFIG_USB_STORAGE=y
+CONFIG_USB_UAS=y
 CONFIG_USB_DWC3=y
 CONFIG_USB_DWC2=y
 CONFIG_USB_DWC2_DUAL_ROLE=y
-- 
2.35.3

